src/utils/errors.ts

```diff
@@ -1,8 +1,9 @@
+type S = string & {}
```

Comment:
In ts < 4, this prevents the const string literal from merging with the `string` type. can be removed to just "string" in ts 4.0+